<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="serial_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="serial_box">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>COM1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM2</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="baud_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="baud_box">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="open_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_button">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭串口</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_button_Led">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭LED</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open_button_Led">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开LED</string>
   </property>
  </widget>
  <widget class="QLabel" name="serial_label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>0</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>接收窗口</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Send">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>871</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="baud_label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>发送窗口</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>591</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SwitchFilepushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="showTextpathEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>370</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
